gdal/scripts/setdevenv.sh

```diff
@@ -14,7 +14,7 @@ if [[ $called == "$0" ]]; then
     exit 1
 fi
 
-GDAL_ROOT=$(realpath -e $(dirname ${BASH_SOURCE[0]})/..)
+GDAL_ROOT=$(readlink -e $(dirname ${BASH_SOURCE[0]})/..)
```

Comment:
are you sure that works on Mac ?

Reply:
Apparently it will, but not with `-e`. :-(